src/nccl_ofi_rdma.c

```diff
@@ -3070,7 +3073,7 @@ static int recv(nccl_net_ofi_recv_comm_t *recv_comm, int n, void **buffers,
 	/* Return request to NCCL */
 	*base_req = (nccl_net_ofi_req_t *)req;
 	/* Increment next_msg_seq_num for next call */
-	++(r_comm->next_msg_seq_num);
+	r_comm->next_msg_seq_num = (r_comm->next_msg_seq_num + 1) % MSG_SEQ_NUM_SIZE;
```

Comment:
Since `MSG_SEQ_NUM_SIZE` is a power of two, would it be more efficient to do this with a bitmask? Same below.